GTFS.DB.PostgreSQL/Collections/PostgreSQLShapesCollection.cs

```diff
@@ -167,6 +168,43 @@ public IEnumerable<Shape> Get(string entityId)
             });
         }
 
+        /// <summary>
+        /// Returns the entities for the given id's.
+        /// </summary>
+        /// <param name="entityIds"></param>
+        /// <returns></returns>
+        public IEnumerable<Shape> Get(List<string> entityIds)
+        {
+            if (entityIds.Count() == 0)
+            {
+                return new List<Shape>();
+            }
+            var sql = new StringBuilder("SELECT id, shape_pt_lat, shape_pt_lon, shape_pt_sequence, shape_dist_traveled FROM shape WHERE FEED_ID = :feed_id AND id = :id0");
```

Comment:
Order returned by PostgreSQL is not deterministic from what I can tell, you should add order by

Reply:
fixed in master